data/bactrian.txt

```diff
@@ -0,0 +1,379 @@
+			`	"We received word just yesterday that just after refueling on Prime, a large pirate fleet entered the system. The Star Queen's escorts were sadly destroyed." He seems to choke up after telling you that. "The pirates then boarded and took control of the Star Queen. We then lost all connection to the vessel, as we have no idea where it could have gone from there." He takes a moment to collect himself after telling you the situation.`
+			choice
+				`	"I would be glad to help. I'll prepare my ship immediately."`
+					accept
+				`	"Why do you seem so sad about the situation?"`
+					goto sad
+				`	"What will I receive in exchange for helping you?"`
+				`	"Why can't you locate it yourself?"`
+					goto locate
+	
+			`	"We will pay you <payment> if you are able to at least find where the Star Queen is, but there may be more reward if you are able to return it."`
+				goto end
+			
+			label sad
+			`	The man pauses before answering you, as if trying to choose his words carefully. "I personally know a lot of the scientists on board that Star Queen, and I just know that someone's going to wind up dead in this situation, and it isn't every day that someone you know gets held captive by pirates. This whole thing is just stressful for me."`
+				goto end
+	
+			label locate
+			`	"The Star Queen had heavy escorts that shouldn't have been easy to destroy, but seeing as how they were we need someone to forge ahead and find the Star Queen now while we assemble a proper rescue fleet. In a situation like this, every minute matters, and if someone like you can find the Star Queen fast then that means that there is a higher chance everyone can come out of this alive."`
+	
+			label end
+			choice
+				`	"Okay, I'm willing to help."`
+					accept
+				`	"Sorry, but this doesn't sound worth it to me."`
+					decline
+
+	on enter Arneb
+		set "found star queen"
+		dialog "As you enter Arneb, you spot the Star Queen landing on Haven, defended by a number of pirate vessels. You will need to report back to the Deep in order to retrieve the Star Queen in one piece."
+	
+	on visit
+		dialog "You have landed on <planet>, but you have not located the Star Queen! Find it somewhere in the Far North."
+	
+	to complete
+		has "found star queen"
+	
+	on complete
+		payment 100000
+
+mission "Bactrian Unlock [0]: Prime"
+	landing
+	source Prime
+	to offer
+		has "Bactrian Unlock [0]: active"
+	on offer
+		conversation
+			`The Star Queen is not in this system, so you attempt to find somebody who may have some information. You eventually find a merchant who claims to have been here when the Star Queen was and that a large pirate fleet jumped in and took the Star Queen hostage before jumping north to Rigel.`
```

Comment:
Depends if it is talking about north as a direction, or the Far North as a [position, location].